packages/flutter/test/material/dropdown_menu_theme_test.dart

```diff
@@ -7,6 +7,28 @@ import 'package:flutter/material.dart';
 import 'package:flutter_test/flutter_test.dart';
 
 void main() {
+  Finder findMenuItemButton(String label) {
```

Comment:
Sometimes I'll create a utils file to share methods like these that are used in multiple test files (e.g. [editable_text_utils.dart](https://github.com/flutter/flutter/blob/master/packages/flutter/test/widgets/editable_text_utils.dart)). Probably not worth it in this case but up to you.

Reply:
Great reminder :pray: 
I have 3 DropdownMenu/MenuAnchor PRs on the work, I will try to figure out if such a utils file is possible.